docs/community/meeting_schedule.md

```diff
@@ -52,10 +52,21 @@ If you are using napari or interested in how napari could be used in your work,
         modal.style.display = "block";
         var eventTitle = eventObj.title.charAt(0).toUpperCase() + eventObj.title.slice(1);
         document.getElementById("details").innerHTML = '<b>' + eventTitle + '</b>' + '<br>' + eventObj.extendedProps.description;
-        //When the user clicks on <span> (x), close the modal
+        // When the user clicks on <span> (x), close the modal
         span.onclick = function() {
           modal.style.display = "none";
         }
+        // When the user clicks anywhere outside of the modal, close it
+        modal.onclick = function(event) {
+          if (event.target.id == "myModal") {
```

Comment:
Should there be a `!` or a `not` somewhere here to trigger when the click is outside the modal target area?
(I've not tested this because on CI the calendar events don't show)

Reply:
Actually no, `MyModal` is the shadow that is overlayed in front of the whole page while the popover itself is the `event`, so this is correct.

Reply:
Ah, amazing! Maybe change the name to "mySneakyBackgroundHidingModalNotTheModalYouReThinking"??? 😜

Reply:
@melissawm so to clarify, this will only work once the new theme is released *and* the constraints are upgraded, right?

Reply:
The logic here is correct. But I prefer to change the name of `myModal` to something more readable. For example,  `eventDetailBackground`

Reply:
@Czaki that would require a theme change, right? I suggest we merge this and then make the parallel changes after releasing 0.4.19...

Reply:
No. It works with current theme:
![obraz](https://github.com/napari/docs/assets/3826210/40b06df1-f8a1-4de5-b141-5a4afa7c9c40)

Reply:
I'm confused: isn't the ID of the background set by the theme? Clearly this is not an arbitrary string, this is a string matching an element of the DOM? Where is the ID of the modal coming from if not the theme?

Reply:
In the expression `if (event.target.id == "myModal")`